src/Console/Command/Compile.php

```diff
@@ -824,12 +830,21 @@ private function logMap(MapFile $fileMapper, CompileLogger $logger): void
         }
     }
 
-    private function logEndBuilding(CompileLogger $logger, SymfonyStyle $io, Box $box, string $path, float $startTime): void
-    {
+    private function logEndBuilding(
+        Configuration $config,
+        CompileLogger $logger,
+        SymfonyStyle $io,
+        Box $box,
+        string $path,
+        float $startTime
```

Comment:
Probably not within the scope of the PR, but is using floats sensible? What is the unit of this (seconds? nanoseconds? picoseconds? hours?...)

Reply:
No idea, worth checking 👍

Reply:
It is micro seconds from `microtime(true)`, so `float` is correct here

Reply:
Ok. Might be worth either commenting that it's microseconds, or wrap it in a VO like `class TimeInMicroseconds` ?